dist/types.d.ts

```diff
@@ -159,7 +159,11 @@ export interface HomeAssistant {
     selectedTheme?: string | null;
     panels: Panels;
     panelUrl: string;
+    /**
+    * @deprecated Use `locale.language` instead
+    */

```

Comment:
I am not sure if this is necessary or if this would even be displayed for users.